contract/models/contract_line.py

```diff
@@ -889,9 +889,9 @@ def cancel(self):
             raise ValidationError(_("Cancel not allowed for this line"))
         for contract in self.mapped("contract_id"):
             lines = self.filtered(lambda l, c=contract: l.contract_id == c)
-            msg = _(
-                "Contract line canceled: %s",
-                "<br/>- ".join(
+            msg = (
+                _("Contract line canceled: %s")
+                % "<br/>- ".join(
```

Comment:
And this syntax is also valid and don't included in the translation. Have you tried?

Reply:
Your right, This is included in the .pot file
I'll remove my changes here